lib/elasticbeanstalk_helper.rb

```diff
@@ -75,4 +75,9 @@ def ssh_command(tier = nil)
   def deploy_command(tier = nil)
     %w(eb deploy) + [environment_name(current_environment_number, tier)]
   end
+
+  def swap_command(tier = nil)
+    current = current_environment_number
+    %W(eb swap #{environment_name(current, tier)} -n #{environment_name(current - 1, tier)})
```

Comment:
```
irb(main):001:0> %W(#{1 + 1} #{'a' * 40})
=> ["2", "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"]
```

Reply:
no idea what you mean here :)

Reply:
I wanted to prove `%W` interpolates `#{...}` stuff.